README.md

```diff
@@ -1,4 +1,4 @@
-# Twitter Clone
+# Twitter Clone - cange for pull request
```

Comment:
I like it how you have done lots of push / pull changes. Makes it easier to review & see the timeline / changes made each time.